### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by edutjie**

### CODE REVIEW

File: app/components/landing-page/GetToKnowUs/index.tsx [hunk 1]
```diff
@@ -5,8 +5,8 @@ import GetToKnowUsCard from './GetToKnowUsCard'
 
 const GetToKnowUs = () => {
 	return (
-		<section className='h-full text-white lg:px-14'>
-			<h2 className='my-24 text-center 2sm:text-left'>
+		<section className='h-full text-white mx-auto max-w-9/10'>
+			<h2 className='my-24 text-center 2md:text-left'>
```
Comment (DJaegerScript) [inline app/components/landing-page/GetToKnowUs/index.tsx:9]: what the "2" preceding "md:text-left" do?

File: app/components/landing-page/MeetTheDevTeam/index.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import Image from 'next/image'
 
 const MeetTheDevTeam = () => {
```
Comment (DJaegerScript) [inline app/components/landing-page/MeetTheDevTeam/index.tsx:7]: Is it meet the dev team, or meet the KA team?

**Review by DJaegerScript — CHANGES_REQUESTED** [review]

File: app/components/landing-page/GetToKnowUs/index.tsx (hunk 1 above)
Comment (edutjie) [reply app/components/landing-page/GetToKnowUs/index.tsx:9]: '2md': '880px', at screen >= 880px the text will be left aligned and < 800px will be center aligned.

File: app/components/landing-page/MeetTheDevTeam/index.tsx (hunk 2 above)
Comment (edutjie) [reply app/components/landing-page/MeetTheDevTeam/index.tsx:7]: Meet The Dev Team is for this button. 



MeetKaTeam is for the ka team page.

File: app/components/landing-page/OurStories/index.tsx [hunk 3]
```diff
@@ -1,32 +1,47 @@
-import React, { useState } from 'react'
+import React, { useEffect, useState } from 'react'
 import { StoryContainer } from '../styles'
-import { contents } from './utils/contents'
+// import { contents } from './utils/contents'
```
Comment (DJaegerScript) [inline app/components/landing-page/OurStories/index.tsx:3]: Is it will be used again?

**Review by DJaegerScript — CHANGES_REQUESTED** [review]

File: app/components/landing-page/MeetTheDevTeam/index.tsx (hunk 2 above)
Comment (DJaegerScript) [reply app/components/landing-page/MeetTheDevTeam/index.tsx:7]: > Meet The Dev Team is for this button.  

How about change it to KA instead of dev?

**Review by DJaegerScript — APPROVED** [review]

**Pull request merged**